specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FlexibleServers.json

```diff
@@ -0,0 +1,983 @@
+            "GeneralPurpose",
+            "MemoryOptimized"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ],
+      "description": "Sku information related properties of a server."
+    },
+    "Storage": {
+      "type": "object",
+      "properties": {
+        "storageSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Max storage allowed for a server."
+        }
+      },
+      "description": "Storage properties of a server"
+    },
+    "AuthConfig": {
+      "type": "object",
+      "properties": {
+        "activeDirectoryAuthEnabled": {
+          "type": "boolean",
+          "description": "If true, Azure Active Directory authentication is enabled."
+        },
+        "passwordAuthEnabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "If true, Password authentication is enabled."
+        },
+        "tenantId": {
+          "type": "string",
+          "default": "",
+          "description": "Tenant id of the server."
+        }
+      },
+      "description": "Authentication configuration properties of a server"
+    },
+    "DataEncryption": {
+      "type": "object",
+      "properties": {
+        "primaryKeyURI": {
+          "type": "string",
+          "description": "URI for the key for data encryption for primary server."
+        },
+        "primaryUserAssignedIdentityId": {
+          "type": "string",
+          "description": "Resource Id for the User assigned identity to be used for data encryption for primary server."
+        },
+        "type": {
+          "type": "string",
+          "description": "Data encryption type to depict if it is System assigned vs Azure Key vault.",
+          "enum": [
+            "SystemAssigned",
+            "AzureKeyVault"
+          ],
+          "x-ms-enum": {
+            "name": "ArmServerKeyType",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Data encryption properties of a server"
+    },
+    "UserIdentity": {
+      "description": "Describes a single user-assigned identity associated with the application.",
+      "type": "object",
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "the object identifier of the Service Principal which this identity represents."
+        },
+        "clientId": {
+          "type": "string",
+          "description": "the client identifier of the Service Principal which this identity represents."
+        }
+      }
+    },
+    "UserAssignedIdentityMap": {
+      "type": "object",
+      "description": "Defines a map that contains user assigned identities.",
+      "additionalProperties": {
+        "$ref": "#/definitions/UserIdentity"
+      }
+    },
+    "Identity": {
+      "type": "object",
```

Comment:
For the lintDiff excetion, it was because this identity definition conflicts with the [common type](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L200), please rename it or use the common type instead

Reply:
Updated name different from common type for Identiy